lib/tasks/argo.rake

```diff
@@ -12,15 +12,19 @@ task :app_version do
 end
 
 def jettywrapper_load_config
-  return Jettywrapper.load_config.merge({:jetty_home => File.expand_path(File.dirname(__FILE__) + '../../../jetty'),:startup_wait => 200})
+  config = {
+    :jetty_home => File.expand_path(File.dirname(__FILE__) + '../../../jetty'),
+    :startup_wait => 240 # seconds - 4 mins
+  }
+  Jettywrapper.load_config.merge(config)
 end
 
 task :default => [:ci]
 
 task :ci do
-  ENV['RAILS_ENV'] = 'test'
+  raise 'NEVER RUN CI OUTSIDE RAILS_ENV=="test"' if ENV['RAILS_ENV'] != 'test'
```

Comment:
1) The entire block could be rewritten to encasulate it in a positive `if ENV['RAILS_ENV'] == 'test'` and not raise an exception at all, but the exception adds emphasis and I like to read an if-dosomething block even if the condition has a `!=` in it ;-)